examples/pytorch/model_zoo/classification.py

```diff
@@ -0,0 +1,87 @@
+from dgl.data import Tox21
+from dgl.data.utils import split_dataset
+from dgl import model_zoo
+import torch
+from torch.nn import BCEWithLogitsLoss
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+
+from utils import Meter, EarlyStopping, collate_molgraphs
```

Comment:
Why do you want to call this staff `Meter`?

Reply:
https://github.com/pytorch/examples/blob/master/imagenet/main.py#L354